Makefile

```
@@ -33,6 +33,7 @@ $(API_SPEC_PATH):
 	rm -rf $(API_CLONE_PATH)
 
 .PHONY: api-generate
+api-generate: old_spec := $(shell ls -1 $(API_SERVICE_PATH)/openapi.*.yaml)
```

```suggestion
old_api_spec := $(shell ls $(API_SERVICE_PATH)/openapi.*.yaml)
```
(no need for the target on the left, renamed to `old_api_spec`, removed `-1`)